web/public/js/controllers/dojo-event-form-controller.js

```diff
@@ -3,6 +3,7 @@
   'use strict';
 
   function getEveryTargetWeekdayInDateRange(startDateTime, endDateTime, targetWeekday, eventType) {
+    endDateTime = moment.utc(endDateTime).add(1, 'days');
```

Comment:
Can you leave a comment explaining why we are adding a magic time of one day here?